client-v2/src/shared/util/articleFragment.ts

```diff
@@ -3,12 +3,25 @@ import { ArticleFragment, ArticleFragmentMeta } from 'shared/types/Collection';
 import pick from 'lodash/pick';
 import cloneDeep from 'lodash/cloneDeep';
 
-const createFragment = (id: string, supporting: string[] = []) => ({
+const createFragment = (
+  id: string,
+  imageHide: boolean = false,
+  imageReplace: boolean = false,
+  imageCutoutReplace: boolean = false,
+  imageCutoutSrc?: String,
+  showByline: boolean = false,
+  showQuotedHeadline: boolean = false
+) => ({
   uuid: v4(),
   id,
   frontPublicationDate: Date.now(),
   meta: {
-    supporting
+    ...(imageHide ? { imageHide } : ''),
+    ...(imageReplace ? { imageReplace } : ''),
+    // ...(imageCutoutReplace ? { imageCutoutReplace } : ''),
+    ...(imageCutoutReplace ? { imageCutoutReplace, imageCutoutSrc } : ''),
```

Comment:
Magic notation 🤯

Reply:
Would it be clearer to spread an empty object here rather than an empty string?

Reply:
That is my instinct but I'm aware my TS foo is still lacking.

Reply:
It does the exact same thing, I think, but OK.